WcaOnRails/config/routes.rb

```diff
@@ -200,6 +200,10 @@
   get 'wca-workbook-assistant' => 'static_pages#wca_workbook_assistant'
   get 'wca-workbook-assistant-versions' => 'static_pages#wca_workbook_assistant_versions'
 
+  scope 'page_data' do
```

Comment:
Is there an exsting "delegates" scope that this can be moved under?
`page_data` is a bit too generic for my taste. If no existing scope really fits, what about `roles_data` or similar?

Reply:
I thought about that, but it's actually returning two data: `delegates` and `canViewDelegateMatters`. So I thought either I should go with some generic name which can accept both or I should go with two different APIs instead of getting both into one. Is it fine if I use two different APIs to get both the data?

Reply:
After thinking again, I think having two different APIs will be better - one to fetch delegate retails, and another to fetch roles of current user. Can I go ahead with two different APIs?